src/br/org/scadabr/vo/usersProfiles/UsersProfileVO.java

```diff
@@ -14,13 +14,14 @@
 import com.serotonin.mango.vo.dataSource.DataSourceVO;
 import com.serotonin.mango.vo.permission.DataPointAccess;
 import org.scada_lts.mango.service.DataSourceService;
+import org.scada_lts.web.dwr.security.NoEscape;
 
 @JsonRemoteEntity
 public class UsersProfileVO implements Cloneable, JsonSerializable {
 
 	public static final String XID_PREFIX = "UP_";
 
-	@JsonRemoteProperty
+	@JsonRemoteProperty @NoEscape
 	private String name;
```

Comment:
As I mentioned, such data must be escaped. Everything returned from the backend must be escaped. The exclude is JavaScript scripts get from the backend, which must be executed by the client, in the browser. These scripts cannot be escaped. This applies, for example, to HTML components in graphical views. This should not apply to scripts executed on the server side, where we deliver unescaped scripts.